package.json

```diff
@@ -59,7 +68,7 @@
     "ember-disable-prototype-extensions": "^1.1.3",
     "ember-load-initializers": "^2.1.2",
     "ember-resolver": "^8.0.3",
-    "ember-source": "~4.9.1",
+    "ember-source": "^4.10.0-beta.2",
```

Comment:
What needs `-beta.2`? I assume the `Owner` types?

Reply:
`Resolver` preview types from beta.1

Reply:
Makes sense. I just landed a PR this morning, and will do the back port for it on Monday, which will make it available (along with other `Owner` types needed) on 4.8 LTS.